File: 2.0/python/me/verifyCallerId.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.me.verifyCallerId(
    callerid='12132000384',
    body={
        'verificationCode': 1234
    }
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/me/verifyCallerId.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("api_login", "api_password");
        $request = $client->verifyCallerId();
        $request->getOperationConfig()->setPathParameters(array("callerid" => "12132000384"));
        $body = '{
                    "verificationCode":"1234"
                 }';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
